docs/install/install-from-packages.md

```diff
@@ -63,25 +52,14 @@ To avoid RAM and disk swap limitations, we recommend either temporarily resizing
     If you get Plone to run on Windows alone, please feel free to document and share your process.
     ```
 
--   [Python](https://www.python.org/downloads/) 3.8, 3.9, or 3.10.
+-   [Python](https://www.python.org/downloads/) 3.8, 3.9, 3.10 or 3.11 at the time of writing. Verify with the [Release notes](https://plone.org/download/releases/).
+-   {pipx}`pipx`
 -   {term}`Cookiecutter`
 -   {term}`nvm`
--   {term}`Node.js` LTS 16.x (see "Update" note)
+-   {term}`Node.js` LTS
 -   {term}`Yeoman`
 -   {term}`Yarn`
 -   {term}`GNU make`
--   {term}`Docker`
```

Comment:
I bring back Docker to the pre-requisites and postpone the removal to the point in time that Docker is optional. Which is wanted because this chapter is about installing Plone for development without Docker at all.